src/Machete.HL7.Tests/Segments/FT.cs

```diff
@@ -0,0 +1,81 @@
+﻿namespace Machete.HL7.Tests.Segments
+{
+    using System;
+    using System.Runtime.Serialization;
+
+
+    [Serializable]
+    public class FT :
```

Comment:
So, in the HL7MacheteTestHarness you have the following code,

            protected override ISchema<TSchema> CreateSchema()
            {
                return Machete.Schema.Factory.CreateHL7<TSchema>(cfg => cfg.AddFromNamespaceContaining<TVersion>());
            }